### PR TITLE
Allow running all services locally

### PR DESCRIPTION
**Pull request by richrace**
### Why

- Allow us to run all services locally

### What 

- Add docker service to allow running from this  docker compose file

### Notes

- run `make run-all` to run all services
- When running all services it's likely you won't see any content in the Worktray. This is due to this line: https://github.com/LBHackney-IT/LBH-IncomeCollection/blob/master/app/controllers/tenancies_controller.rb#L58

### CODE REVIEW

**Comment by kjdchapman** [comment]
I'm taking a look at this, but it's taking a good while to run locally!

**Comment by kjdchapman** [comment]
If I'm understanding right, this has a dependency on other git repos being on the host machine?

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,13 @@
+INCOME_API_DIR  ?= ../lbh-income-api
+UHSIM_DIR       ?= ../universal-housing-simulator
+TENANCY_API_DIR ?= ../LBHTenancyAPI
+
+ifneq (, $(wildcard ${INCOME_API_DIR}/.env))
```
Comment (bjpirt) [inline Makefile]: I did wonder whether we should do this or not - loading in the env variables from the .env file means that you might end up pointing at staging APIs etc, when we can be pretty confident that you're actually trying to run all services locally when running this script (I know I added this initially!)

**Comment by richrace** [comment]
> If I'm understanding right, this has a dependency on other git repos being on the host machine?

Correct

File: spec/lib/hackney/income/tenancy_gateway_spec.rb [hunk 2]
```diff
@@ -282,7 +282,10 @@
               service: "¤#{Faker::Number.decimal(2)}",
               other_charge: "¤#{Faker::Number.decimal(2)}",
               current_arrears_agreement_status: Faker::Lorem.characters(3),
-              current_balance: "¤#{Faker::Number.decimal(2)}",
+              current_balance: {
+                value: "¤#{Faker::Number.decimal(2)}",
+                currency_code: 'GBP'
+              },
```
Comment (elena-vi) [inline spec/lib/hackney/income/tenancy_gateway_spec.rb]: ermm, has this been changed in the api?
also not comfortable with this change being made here

Comment (richrace) [reply spec/lib/hackney/income/tenancy_gateway_spec.rb]: This is not a change to the API.

This code was changed: https://github.com/LBHackney-IT/LBH-IncomeCollection/pull/242/files#diff-63b5d8c3f0146e87d16f7d8d02d32203L99
which broke this test.

File: Makefile (hunk 1 above)
Comment (richrace) [reply Makefile]: I can remove loading the `.env` so there's no doubt.

File: spec/lib/hackney/income/tenancy_gateway_spec.rb (hunk 2 above)
Comment (richrace) [reply spec/lib/hackney/income/tenancy_gateway_spec.rb]: Had an in-person chat around this.

**Pull request merged**